### ISSUE TITLE
(Question) Error while building the project in Ubuntu

### ISSUE THREAD

**Issue by shivangdoshi** [wg-perception/PartsBasedDetector#25]
Hey all,
Can you help me out a bit. I am trying to run this project and am following the given cmake routine but i am facing following error-

CMake Error: The following variables are used in this project, but they are set to NOTFOUND.
Please set them or make sure they are set and tested correctly in the CMake files:
Boost_INCLUDE_DIR (ADVANCED)
   used as include directory in directory /var/www/ccpp/PartsBasedDetector-master
   used as include directory in directory /var/www/ccpp/PartsBasedDetector-master/src

-- Configuring incomplete, errors occurred!

I've also reinstalled Boost library but it is still the same.
Basically I want get this project running but I am not able to do that. Can anyone help with an overview.

Any help is much appreciated.

Thanks !
